src/main/java/roomescape/exception/ErrorResult.java

```diff
@@ -0,0 +1,20 @@
+package roomescape.exception;
+
+public class ErrorResult {
+
+    private String code;
+    private String message;
```

Comment:
불변이 가능한 곳은 `final`을 잊지 말아주세요!
```suggestion
    private final String code;
    private final String message;
```

\+ 얘도 데이터 객체와 다르지 않아서 record도 적합해 보이네요